### ISSUE TITLE
Daniel has 3 syllabes, not 2

### ISSUE THREAD

**Issue by daberlin** [heiglandreas/TextStatistics#6]

**Comment by heiglandreas** [comment]
To count Syllables we first need to create them. As there is not one international single perfect way to get syllables from a word or text I choose to use the way that is used to hyphenate words as that mostly resembles syllables and there are libraries readily available. I know perfectly well that that that is not perfectly accurate but from the researches I did I found that all other ways aren't perfect as well and there are always failures. The TeX-Hyphenation algorithm is - besides the Dieckmann-Algorithm - one of the best compromises between size, speed and accuracy and as it's also OSS, I decided to choose that one to create the syllables knowing perfectly well that the results aren't 100% perfect. But as the syllables are intended to calculate text-metrics a certain inaccuracy is allowable. 

According to the TeX-Hyphenation-algorithms that are used to create the syllables ```Daniel``` should only be hyphenated ```Da-niel```. As that Algorithm is (as every other hyphenation-algorithm) not perfect that might be one of those rare cases where there are more options than the algorithm knows of.

So TL;DR - Yes, but it's as it is, there's currently not an easy way to change that. If you know of one feel free to tell me!
